src/OVAL/oval_enumerations.c

```diff
@@ -761,6 +779,17 @@ const char *oval_subtype_to_str(oval_subtype_t subtype)
 	return oval_subtype_get_text(subtype);
 }
 
+oval_subtype_t oval_subtype_from_str(const char *str)
+{
```

Comment:
This function cannot work and there was a good reason why there was no such function :smile:: One string can map to multiple OVAL subtypes. For example `"interface"` can map to `OVAL_IOS_INTERFACE` or `OVAL_UNIX_INTERFACE` or `OVAL_WINDOWS_INTERFACE`.